conf/application.conf

```diff
@@ -14,6 +14,11 @@
 
 include "frontend.conf"
 
+contact-frontend {
+  host = "http://localhost:9250"
+  serviceId = "MTDIT"
```

Comment:
We should make this unique to this service ideally.
I don't think it needs approval by anyone as long as we are pretty sure it's unique.
So... `ITSC`?
Might want to do a github search for the same first though

Reply:
As far as I can tell, ITSC should be fine, couldn't find it used anywhere else